controllers/admissionpolicy_controller_test.go

```diff
@@ -201,86 +204,91 @@ var _ = Describe("AdmissionPolicy controller", func() {
+			).To(Succeed())
+
+			By("reconciling the MutatingWebhookConfiguration to its original state")
+			Eventually(func() (*admissionregistrationv1.MutatingWebhookConfiguration, error) {
+				return getTestMutatingWebhookConfiguration(fmt.Sprintf("namespaced-%s-%s", policyNamespace, policyName))
+			}, timeout, pollInterval).Should(
+				And(
+					HaveField("Labels", Equal(originalMutatingWebhookConfiguration.Labels)),
+					HaveField("Annotations", Equal(originalMutatingWebhookConfiguration.Annotations)),
+					HaveField("Webhooks", Equal(originalMutatingWebhookConfiguration.Webhooks)),
+				),
+			)
 
-		It("should set the policy status to unscheduled", func() {
+			// simulate unitialized labels and annotation maps (behaviour of Kubewarden <= 1.9.0), or user change
+			By("by setting the MutatingWebhookConfiguration labels and annotation to nil")
+			mutatingWebhookConfiguration, err := getTestMutatingWebhookConfiguration(fmt.Sprintf("namespaced-%s-%s", policyNamespace, policyName))
+			Expect(err).ToNot(HaveOccurred())
+			originalMutatingWebhookConfiguration = mutatingWebhookConfiguration.DeepCopy()
+			mutatingWebhookConfiguration.Labels = nil
+			mutatingWebhookConfiguration.Annotations = nil
 			Expect(
-				k8sClient.Create(ctx, admissionPolicyFactory(policyName, policyNamespace, "", false)),
-			).To(haveSucceededOrAlreadyExisted())
+				k8sClient.Update(ctx, mutatingWebhookConfiguration),
+			).To(Succeed())
 
-			Eventually(func() (*policiesv1.AdmissionPolicy, error) {
-				return getTestAdmissionPolicy(policyNamespace, policyName)
-			}, 30*time.Second, 250*time.Millisecond).Should(
-				HaveField("Status.PolicyStatus", Equal(policiesv1.PolicyStatusUnscheduled)),
+			By("reconciling the MutatingWebhookConfiguration to its original state")
+			Eventually(func() (*admissionregistrationv1.MutatingWebhookConfiguration, error) {
+				return getTestMutatingWebhookConfiguration(fmt.Sprintf("namespaced-%s-%s", policyNamespace, policyName))
+			}, timeout, pollInterval).Should(
+				And(
+					HaveField("Labels", Equal(originalMutatingWebhookConfiguration.Labels)),
+					HaveField("Annotations", Equal(originalMutatingWebhookConfiguration.Annotations)),
+					HaveField("Webhooks", Equal(originalMutatingWebhookConfiguration.Webhooks)),
+				),
 			)
 		})
+
 	})
 
-	When("creating an AdmissionPolicy with a PolicyServer assigned but not running yet", func() {
-		var (
-			policyName       = newName("scheduled-policy")
-			policyServerName = newName("policy-server")
+	It("should set policy status to unscheduled when creating an AdmissionPolicy without a PolicyServer assigned", func() {
+		policyName := newName("unscheduled-policy")
+		Expect(
+			k8sClient.Create(ctx, admissionPolicyFactory(policyName, policyNamespace, "", false)),
+		).To(haveSucceededOrAlreadyExisted())
+
+		Eventually(func() (*policiesv1.AdmissionPolicy, error) {
+			return getTestAdmissionPolicy(policyNamespace, policyName)
+		}, 30*time.Second, 250*time.Millisecond).Should(
```

Comment:
here we seem to be able to get away with the consts `timeout`, `pollInterval`, they are bigger.